client/dashboard/.eslintrc.js

```diff
@@ -25,6 +25,10 @@ module.exports = {
 							'calypso/assets/*',
 							'!calypso/assets/icons',
 							'!calypso/assets/images',
+							// Allowed: calypso/components/marked-lines
+							'!calypso/components',
+							'calypso/components/*',
+							'!calypso/components/marked-lines',
```

Comment:
I'm a bit worried about the precedent this sets. Isn't it preferable to do the other way around, and move the updated component to `client/dashboard/` and then import the "new" component from the legacy code?

The reasoning I'm following is that when a new implementation imports legacy code, it makes deprecating that legacy code harder later: "new code importing legacy code becomes legacy code". By contrast, porting the component first and importing it from the legacy side avoids creating this dependency.

Reply:
Thanks for sharing this! I understand that the idea of restricting components outside the Hosting Dashboard by default is to ensure we are properly informed about what we are importing.

I reached out to the folks here (p1758243310235129-slack-C0982082MSR) to ask for confirmation before importing it. It turned out to be an interesting exercise because we realized we needed to update the CSS styles and remove the lodash dependency to migrate it properly, so I went ahead and did that. :D 

Doing it the other way around (moving it to HD and importing it from the legacy code) doesn't sound too crazy neither, @Automattic/lego do you have any thoughts on that?